webv2/summary/summary_service.go

```diff
@@ -8,39 +8,39 @@ import (
 // getSummary returns table wise summary of conversion.
 func getSummary() map[string]ConversionSummary {
 	sessionState := session.GetSessionState()
-	reports := reports.AnalyzeTables(sessionState.Conv, nil)
+	Reports := reports.AnalyzeTables(sessionState.Conv, nil)
```

Comment:
Why do we need Reports to be public?

Reply:
Yes we don't need it to be public, corrected!